src/app/plan/components/Wizard/WizardComponent.tsx

```diff
@@ -207,10 +247,15 @@ const WizardComponent = (props: IOtherProps & FormikProps<IFormValues>) => {
       errors,
       touched,
       currentPlanStatus,
+      migHookList,
```

Comment:
This big useEffect is wild, I might take a closer look at it later 😆

Reply:
Yeah It could use another look. Initially our entire wizard was rapidly rerendering due to background polling. Now that the background polling is under control, this is another candidate for refactor that we never got around to.

Reply:
Gotcha. If we still need to guard against rerendering here, we could use `React.memo` to make it only render if certain props change. (that's the new functional equivalent to the old React class `shouldComponentUpdate` lifecycle method). https://reactjs.org/docs/hooks-faq.html#how-do-i-implement-shouldcomponentupdate